OpenEdXMobile/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -52,6 +52,7 @@
 import static org.junit.Assert.assertNotNull;
 import static org.mockito.Mockito.when;
 
+@Ignore("Include this test once we have migrate to the androidX")
```

Comment:
- `Include this test once we have migrated to the androidX`
- Also, add a TODO leading to the ticket where we will be doing the migration to androidX
- The ticket should also mention in its acceptance criteria to look for all such TODOs and fix them